src/actions/postAction.js

```diff
@@ -1,95 +1,101 @@
-import axios from 'axios';
-import { errorHandler } from '../utils/errorHandler';
-import { setRequestStatus } from '../utils/setRequestStatus';
-import { GET_ALL_POSTS, GET_ALL_PINNED_POSTS, GET_SINGLE_POST } from './types';
-import { BASE_URL } from './baseApi'
+import axios from "axios";
+import { errorHandler } from "../utils/errorHandler";
+import { setRequestStatus } from "../utils/setRequestStatus";
+import { GET_ALL_POSTS, GET_ALL_PINNED_POSTS, GET_SINGLE_POST } from "./types";
+import { BASE_URL } from "./baseApi";
 
 // GET ALL POSTS
 export const getAllPosts = (pagination = 10, page = 1) => async (dispatch) => {
   try {
-    const res = await axios.get(`${BASE_URL}/post/all_posts?pagination=${pagination}&page=${page}`)
-    dispatch(setRequestStatus(false))
-    if(res.status === 200) {
-      dispatch(setRequestStatus(true))
-      console.log('all posts ', res.data.posts)
+    const res = await axios.get(
```

Comment:
@AuraOfDivinity Please remove these unnecessary format changes, this is pretty annoying issue